src/pages/token/rewards/UnbondingPanels.tsx

```diff
@@ -0,0 +1,119 @@
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AssetIcon } from '@/components/AssetIcon';
+import { Output, OutputType } from '@/components/Output';
+import { Panel } from '@/components/Panel';
+import { ValidatorFaviconIcon } from '@/components/ValidatorName';
+
+import { getUnbondingDelegations } from '@/state/accountSelectors';
+import { useAppSelector } from '@/state/appTypes';
+
+export const UnbondingPanels = () => {
+  const stringGetter = useStringGetter();
+  const unbondingDelegations = useAppSelector(getUnbondingDelegations, shallowEqual);
+  const { unbondingValidators } = useStakingValidator() ?? {};
+  const { chainTokenLabel, chainTokenDecimals } = useTokenConfigs();
+
+  if (!unbondingDelegations?.length) {
+    return null;
+  }
+
+  return (
+    <$Container>
+      {unbondingDelegations.map((delegation) => {
+        const completionDate = new Date(delegation.completionTime).getTime();
+        const currentDate = new Date().getTime();
+        const timeDifference = completionDate - currentDate;
+
+        const dayDifference = Math.floor(timeDifference / (1000 * 60 * 60 * 24));
+
+        return (
+          <Panel
+            key={delegation.validator}
+            slotHeaderContent={
+              <>
+                <$Title>
+                  {stringGetter({
+                    key: STRING_KEYS.UNSTAKING_FROM,
+                    params: {
+                      VALIDATOR:
+                        unbondingValidators?.[delegation.validator]?.[0]?.description?.moniker,
+                    },
+                  })}
+                </$Title>
+                <$ValidatorFaviconIcon
+                  url={unbondingValidators?.[delegation.validator]?.[0]?.description?.website}
+                  fallbackText={
+                    unbondingValidators?.[delegation.validator]?.[0]?.description?.moniker
+                  }
+                />
+              </>
+            }
+          >
+            <$Content>
+              <$Balance
+                type={OutputType.Asset}
+                value={formatUnits(BigInt(delegation.balance), chainTokenDecimals)}
+                slotRight={<$AssetIcon symbol={chainTokenLabel} />}
+              />
+              <$Footer>
+                {stringGetter({
+                  key: STRING_KEYS.AVAILABLE_IN_DAYS,
+                  params: {
+                    DAYS: dayDifference,
+                  },
+                })}
+              </$Footer>
+            </$Content>
+          </Panel>
+        );
+      })}
+    </$Container>
+  );
+};
+
+const $Container = styled.div`
+  ${layoutMixins.flexColumn}
+  gap: 1.5rem;
+  flex: 1;
+`;
+
+const $Title = styled.h3`
+  ${layoutMixins.inlineRow}
+  margin-bottom: -1.5rem;
+
+  font: var(--font-medium-book);
+  color: var(--color-text-1);
+`;
+
+const $Content = styled.div`
+  ${layoutMixins.flexColumn}
+  gap: 1rem;
+`;
+
+const $ValidatorFaviconIcon = styled(ValidatorFaviconIcon)`
+  margin-bottom: -1.5rem;
```

Comment:
instead of putting the same `margin-bottom` on the `Icon` and `Title`, maybe we could wrap the two in a parent div and put the margin on that?

Reply:
That was my original approach, but ended up fighting the css a bit and couldn't get the width right 😭 I figured this was ok and the Header in Panel is using space-between, which was helpful.